pac4j-cas/src/main/java/org/pac4j/cas/credentials/authenticator/CasAuthenticator.java

```diff
@@ -90,9 +90,9 @@ public void validate(final TokenCredentials credentials, final WebContext contex
             if (configuration.getProxyReceptor() != null) {
                 profile = getProfileDefinition().newProfile(principal, configuration.getProxyReceptor());
                 profile.setId(ProfileHelper.sanitizeIdentifier(profile, id));
-                getProfileDefinition().convertAndAdd(profile, newAttributes);
+                getProfileDefinition().convertAndAdd(profile, newAttributes, null);
```

Comment:
Instead of `null`, we should use the authentication attributes retrieved via `assertion.getAttributes()`.

Reply:
Aha, OK, I'll have a look. I generally adapted all the calls by using profile attributes only to make it compile and act exactly like now (without authentication attributes). But you are right, we can improve it by using the authentication attributes where possible, since they are already available.